### PR TITLE
new team members

### PR DESCRIPTION
**Pull request by titusfortner**
@Rodney-QAGeek @abaird @thekiiingbob, et al. Go ahead and make sure I have everything correct in here, or request changes (minor or major). I'll plan on posting tomorrow with whatever feedback I've gotten by then.

### CODE REVIEW

**Comment by abaird** [comment]
:thumbsup:

File: _contributors/baird_alan.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+name: Alan Baird
+twitter: bluegrasscoder
+location: Austin, TX
+website: http://twitter.com/bluegrasscoder
```
Comment (abaird) [inline _contributors/baird_alan.md]: I don't have anything here yet but you might as well use this: http://abaird.github.io/

**Comment by abaird** [comment]
Would it be better to use the gravatar URL for pics? The URL in Ruby should be ```"https://gravatar.com/avatar/#{Digest::MD5.new.update('abaird@bairdsnet.net').hexdigest)}"```
Or you can just use `https://gravatar.com/avatar/bfc3ff7f71c742ece721e682214ddbb9` for me.

(more info on gravatars if you don't have the link handy: https://en.gravatar.com/site/implement/)

**Comment by thekiiingbob** [comment]
This looks good to me! @abaird we can look into gravatar support. Would that require anyone we have on the list to have a gravatar?

**Comment by Rodney-QAGeek** [comment]
:shipit:

File: _contributors/baird_alan.md (hunk 1 above)
Comment (titusfortner) [inline _contributors/baird_alan.md]: updated

**Comment by titusfortner** [comment]
@abaird I'm not sure if what we have works interchangeably with gravatar, but it's a good idea. I'll leave it to @thekiiingbob to decide if he wants to implement it. :-D

**Comment by abaird** [comment]
@thekiiingbob - it depends on how you implement it. I just figured using the gravatar URL for anyone that had it would be easier than managing the actual picture. But, if this is a pain to implement then it's no big deal to me.

**Pull request merged**